views/components/organisms/o-carousel.scss

```diff
@@ -0,0 +1,102 @@
+@import "../../../src/sass/variables";
+@import "../../../src/sass/common";
+@import "../atoms/a-img";
+
+.slideshow-container {
+  position: relative;
+  margin: auto;
+  box-shadow: 0 14px 6px -7px #eeeeee;
+  .prev, .next {
+    @extend .cursor-link;
+    position: absolute;
```

Comment:
Its always helpful if you create classes for such properties so that it can be directly used on the DOM helps to reduce CSS size

Reply:
Sure. I have removed the classes from css and added them in DOM